packages/glenstack.com/src/components/landing/footer.tsx

```diff
@@ -190,9 +190,28 @@ export const Footer: FC<{ noCTA?: boolean }> = ({ noCTA = false }) => {
           </div>
         </div>
         <div className="mt-12 border-t border-gray-200 pt-8">
-          <p className="text-base text-gray-400 xl:text-center">
-            &copy; 2021 Glenstack Ltd. All rights reserved.
-          </p>
+          <div className="w-full text-sm font-light text-gray-400 lg:flex lg:justify-between lg:items-center">
+            <span className="block mb-2 lg:w-1/3 lg:my-0">
+              ©2021 Glenstack Ltd. Made worldwide.
+            </span>{" "}
+            <ul className="lg:w-1/3 lg:text-center">
+              <li className="inline-block mb-0 mr-4 last:mr-0">
+                {/* <a href="/privacy/">Privacy Policy</a>{" "} */}
```

Comment:
Don't these links exist already above in the Legal section? Can we just take them out here?

Reply:
Forgot what I did there. Let's see 😛